packages/select/src/components/multi-select/multiSelect.tsx

```diff
@@ -332,7 +356,15 @@ export class MultiSelect<T> extends AbstractPureComponent<MultiSelectProps<T>, M
 
     // Popover interaction kind is CLICK, so this only handles click events.
     // Note that we defer to the next animation frame in order to get the latest activeElement
-    private handlePopoverInteraction = (nextOpenState: boolean, evt?: React.SyntheticEvent<HTMLElement>) =>
+    private handlePopoverInteraction = (nextOpenState: boolean, evt?: React.SyntheticEvent<HTMLElement>) => {
+        if (this.props.customTarget != null) {
+            // Cannot rely on input to determine popover state as the input will be inside the Popover
+            // if customTarget is provided
```

Comment:
```suggestion
```
I think this comment only makes sense right now if you assume relying on the input is the "correct" way - I don't think we need to explain handling this separately. If anything the input focus handling could be explained, but I'd rather not just slap a comment on there now since we may miss some nuance about why it was done that way.

Reply:
sounds good!